### PR TITLE
{math}[foss/2022a] magma v2.6.2

### PR DESCRIPTION
**Pull request by casparvl**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
gcn1 - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz, 4 x NVIDIA NVIDIA A100-SXM4-40GB, 515.43.04, Python 3.6.8
See https://gist.github.com/3312e09c2f71d6af7079e385abde7284 for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
gcn1 - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz, 4 x NVIDIA NVIDIA A100-SXM4-40GB, 515.43.04, Python 3.6.8
See https://gist.github.com/bace94cc78d56c56eb4e5d5c86b12d46 for a full test report.

File: easybuild/easyconfigs/m/magma/magma-2.6.2-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# default CUDA compute capabilities to use (override via --cuda-compute-capabilities)
+cuda_compute_capabilities = ['3.5', '5.0', '6.0', '7.0', '7.5', '8.0', '8.6']
+
+# make sure both static and shared libs are built
+configopts = [
+    '-DBUILD_SHARED_LIBS=%s -DGPU_TARGET="%%(cuda_sm_space_sep)s" ' % local_shared for local_shared in ('ON', 'OFF')
+]
+
+sanity_check_paths = {
+    'files': ['lib/libmagma.so', 'lib/libmagma.a'],
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/m/magma/magma-2.6.2-foss-2022a-CUDA-11.7.0.eb:39]:
```suggestion
    'files': ['lib/libmagma.%s' % SHLIB_EXT, 'lib/libmagma.a'],
```

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Review by Micket — COMMENTED** [review]
we've missed this in the 2021 toolchains as well, but i think this should depend on UCX-CUDA as well. 

This issue hasn't surfaced because so far everything that depends on magma has picked up UCX-CUDA anyway

**Comment by akesandgren** [comment]
@casparvl I second the comment from @Micket, it should have a dep for UCX-CUDA

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/magma/magma-2.6.2-foss-2022a-CUDA-11.7.0.eb (hunk 1 above)
Comment (casparvl) [reply easybuild/easyconfigs/m/magma/magma-2.6.2-foss-2022a-CUDA-11.7.0.eb]: Done

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera41-1 - Linux Rocky Linux 8.5, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, 1 x NVIDIA Tesla T4, 510.73.08, Python 3.6.8
See https://gist.github.com/af99206b87b36eeb139af1ab67afdf07 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @casparvl!

**Pull request merged**